lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -543,10 +549,12 @@ export class UserAgentApplication {
             if (interactionType === Constants.interactionTypeRedirect) {
                 if (!isLoginCall) {
                     this.cacheStorage.setItem(`${TemporaryCacheKeys.STATE_ACQ_TOKEN}${Constants.resourceDelimiter}${request.state}`, serverAuthenticationRequest.state, this.inCookie);
+                    this.logger.verbose("State cached for redirect");
                 }
```

Comment:
Can we add an `else` and a logger statement for when `isLoginCall` is true?

Reply:
Added. Let me know if you think the log message needs to be changed